android-junit5/src/test/groovy/de/mannodermaus/gradle/plugins/android_junit5/AndroidJUnitPlatformSpec.groovy

```diff
@@ -18,7 +18,7 @@ import spock.lang.Specification
 abstract class AndroidJUnitPlatformSpec extends Specification {
 
     protected static final COMPILE_SDK = 26
-    protected static final BUILD_TOOLS = "26.0.1"
+    protected static final BUILD_TOOLS = "26.0.2"
```

Comment:
Please also update `.travis.yml` to pull in this version of the Build Tools, otherwise CI won't be able to accept the license and fail.